File: Java/DesignPatterns/src/main/java/designPatterns/behavioral/command/MainCommand.java
package designPatterns.behavioral.command;

public class MainCommand {
  public static void main(String[] args) {
    Receiver receiver = new Receiver();
    Command commandA = new ConcreteCommand.A(receiver);
    Command commandB = new ConcreteCommand.B(receiver);
    Invoker invoker = new Invoker();

    invoker.setCommand(commandA);
    System.out.println(invoker.invokeExecute());

    invoker.setCommand(commandB);
    System.out.println(invoker.invokeExecute());
  }
}

File: Kotlin/DesignPatterns/src/main/kotlin/behavioral/command/MainCommand.kt
package behavioral.command

fun main() {
  var receiver: Receiver = Receiver();
  var commandA: Command = ConcreteCommand.A(receiver)
  var commandB: Command = ConcreteCommand.B(receiver)
  var invoker: Invoker = Invoker()

  invoker.command = commandA
  println(invoker.invokeExecute())

  invoker.command = commandB
  println(invoker.invokeExecute())
}